test/parallel/test-tls-max-send-fragment.js

```diff
@@ -33,10 +33,39 @@ const buf = Buffer.allocUnsafe(10000);
 let received = 0;
 const maxChunk = 768;
 
+const invalidArgumentError = {
+  name: 'TypeError',
+  code: 'ERR_INVALID_ARG_TYPE'
+};
+
+const rangeError = {
+  name: 'RangeError',
+  code: 'ERR_OUT_OF_RANGE'
+};
+
+const typeError = {
+  name: 'TypeError'
+};
+
 const server = tls.createServer({
   key: fixtures.readKey('agent1-key.pem'),
   cert: fixtures.readKey('agent1-cert.pem')
 }, function(c) {
+
+  // No size is passed
+  assert.throws(() => c.setMaxSendFragment(), invalidArgumentError);
+
+  // Invalid arg is passed
+  [null, undefined, '', {}, false, true].forEach((arg) => {
```

Comment:
```suggestion
  [null, undefined, '', {}, false, true, Symbol(), []].forEach((arg) => {
```

Reply:
@aduh95 passing Symbol does not return same error code

Reply:
@aduh95 Is there a way to group several expected error types?

Reply:
Passing `Symbol` should produce the same error, there’s no reason to treat it differently.

Reply:
for some reason Symbol behaves differently ... its a type error without a code

Reply:
Oh right, doing `Symbol() | 0` throws a `TypeError` and that's what `validateInt32` is doing, I didn't know that. In this case, the current code looks good to me :+1: